definitions/divorce/json/CaseEvent/CaseEvent-general-referral-nonprod.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "LiveFrom": "13/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "generalReferral",
+    "Name": "General referral",
+    "Description": "General referral",
+    "DisplayOrder": 16,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/general-referral",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
```

Comment:
`RetriesTimeoutURLAboutToSubmitEvent` is ignored by CCD code. You don't need it. It doesn't help, it doesn't harm. It's just useless :)

Reply:
Ok, i'd leave it in, just incase it becomes useful one day at 5pm Friday evening 😜

Reply:
It won't :)